Logging/RockLib.Logging.Analyzers.Package/RockLib.Logging.Analyzers.Package.csproj

```diff
@@ -31,7 +31,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\RockLib.Logging.Analyzers.CodeFixes\RockLib.Logging.Analyzers.CodeFixes.csproj" />
```

Comment:
Nit: The other csproj files had their indentation adjusted when they were touched, it might be good to adjust this for consistency across the analyzers.

Reply:
Actually, the Package and CodeFix folder should've been deleted, as everything is merged into the Analyzer project now. So....I'll delete those folders :)